javascript/grid-ui/src/components/RunningSessions/RunningSessions.tsx

```diff
@@ -283,6 +293,18 @@ function RunningSessions (props) {
                 />
                 <TableBody>
                   {stableSort(rows, getComparator(order, orderBy))
+                    .filter((session) => {
+                      if (searchFilter === '') {
+                        // don't filter anything
+                        return true
+                      }
+                      const [filterField, filterItem] = searchFilter.split('=')
+                      if (filterField.startsWith('capabilities,')) {
+                        const capabilityID = filterField.split(',')[1]
+                        return (JSON.parse(session.capabilities as string) as object)[capabilityID] === filterItem
+                      }
+                      return session[filterField] === filterItem
+                    })
```

Comment:
I found it misleading when typing in the search a valid keyword, such as `browserName`, then all rows get filtered and nothing is shown. That is why I find the filter logic a bit too complex. It is not really necessary to introduce a query language for a quick search for now. 

Hence my suggestion in the code change, a simple filter based on the entered string and all available capabilities. After that we can wait for feedback from the users.

Reply:
@diemol Thank you for reviewing it. I understand that this might be a little complex however I was hoping we could keep some of the logic. My use case of the grid has 100s of tests running at the same time and they all have lots of capabilties such as `se:name` and other custom capabilities `se:some_id` that might not work great with a simpler search.

 Here's an example:
```json
{
  // capability 
  {
    "se:name": "Don't test Chrome - Sample Test"
    "se:some_id": "no-chrome"
  },
  "browserName": "edge"
}
```
If I were to search for `chrome` in the search it would show me this example from above which doesn't really help me but having the abiltiy to do `browserName=chrome` I can search specific capabilties.


As an alternative, maybe we can do something like this which will allow users to just type random stuff and filter as long as it doesn't have an `=` symbol and for more complex use cases we can do the query-ish syntax.
```js
.filter((session) => {
  if (searchFilter === '') {
    // don't filter anything
    return true
  }

  // NEW CODE
  if (!searchFilter.includes('=')) {
    return (session.capabilities as string).toLowerCase()
                               .indexOf(searchFilter.toLowerCase()) > -1
  }
  // NEW CODE - END

  const [filterField, filterItem] = searchFilter.split('=')
  if (filterField.startsWith('capabilities,')) {
    const capabilityID = filterField.split(',')[1]
    return (JSON.parse(session.capabilities as string) as object)[capabilityID] === filterItem
  }
  return session[filterField] === filterItem
})
```


Please let me know what you think and if this alternative is not desired I can remove my code.

Reply:
OK, that sounds like a good alternative. Looking forward to the changes!